ddev/ddev
# refactor: standardize suggested commands for container log, fixes #5899

## The Issue

- #5899

## How This PR Solves The Issue

There are three functions in the code `ContainerWait`, `ContainersWait`, `ContainerWaitLog` that use different methods to provide the user with more detailed information about what to do with the failed containers.

This PR tries to provide the same output for all functions, giving the user as much information as possible about what to check.

## Manual Testing Instructions

Try breaking project containers on `ddev start` and see what happens.

I don't really know how to break them, so I've only really tested a timeout scenario using:

```
echo 'RUN chmod 000 /run/php' > .ddev/web-build/Dockerfile
```

## Automated Testing Overview



## Related Issue Link(s)

## Release/Deployment Notes
